### DISCUSSION TITLE
installed ,but how /where can i get any plugin?

### DISCUSSION THREAD

**Discussion by lingdu-xu** [emqx/neuron#621] [Q&A]
installed successfully,but i can not see any plugin to use,is it right pls
![捕获63](https://user-images.githubusercontent.com/42382041/177510773-f4092edb-96c4-4d05-85d5-e5d08da40a4e.PNG)
?

**Comment by yuxi311** [comment]
Method one: Update latest code or view the .so file under build/plugins, click "Add plugin" in the upper right corner of the page and fill in the .so file name, such as libplugin-mqtt.so.
Method two: Fill in neuron/default_plugins.json, the existing .so file under neuron/build/plugins.
